src/src/components/MegaMenuNavbar/index.js

```diff
@@ -79,13 +79,13 @@ const MegaMenuNavbar = () => {
     <nav className="navbar navbar--fixed-top">
       <div className="navbar__inner">
         <div className="navbar__items">
-          <Link to="/" className="navbar__brand">
+          <a href="https://peridio.com" className="navbar__brand" target="_blank" rel="noopener noreferrer">
             <img src={`/${navbar.logo.src}`} alt={navbar.logo.alt} className="navbar__logo" />
-          </Link>
+          </a>
           <div className="mega-menu-container">
-            <Link to="/dev-center" className="navbar__item navbar__link">
-              Get Started
-            </Link>
+            <a href="https://docs.peridio.com" className="navbar__item navbar__link">
+              Developer Home
+            </a>
```

Comment:
I believe we would still want to use https://docusaurus.io/docs/docusaurus-core#link for external links.

Doing that for example automatically handles `target="_blank" rel="noopener noreferrer"` for you so you don't have to provide it.

The habit of using `<Link>` always is good because its what enables Docusaurus to validate broken links before publishing.